.github/workflows/ci.yml

```diff
@@ -165,6 +165,14 @@ jobs:
               ../build/examples/trixi_controller_data_f . libelixir_p4est2d_dgsem_euler_sedov.jl
         env:
           LIBTRIXI_DEBUG: all
+      - name: Run julia tests with coverage
+        if: ${{ matrix.test_type == 'coverage' }}
+        run: |
+          cd libtrixi-julia
+          JULIA_DEPOT_PATH=~/.julia \
+              julia --code-coverage=user --code-coverage=@.. --project=. ../test/runtests.jl
```

Comment:
This generates `*.cov` files.

How to proceed? Merge them (as in `Trixi.jl`), export `lcov.info`, and merge it somehow with other one, generated from C-code?